canonical-web-and-design/charmhub.io
# 1873 use store template for search results

## Done
- _Use store template for search results_
- _Remove hero if there is a query parameter in the url_

## How to QA
- Go to https://charmhub-io-833.demos.haus/?q=grafana,kube and see there are some results and no hero
- Go to https://charmhub-io-833.demos.haus and see there is a hero

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1873
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1869

## Screenshots
[if relevant, include a screenshot]
